src/base/preferences.h

```diff
@@ -98,6 +99,10 @@ class Preferences : public QObject
     // General options
     QString getLocale() const;
     void setLocale(const QString &locale);
+    bool useCustomUITheme();
+    void setUseCustomUITheme(bool);
+    QString customUIThemePath();
+    void setCustomUIThemePath(const QString &);
```

Comment:
you should put in the var name... why do I even need to point this (and many other small coding style defectives) out... this is easily seen not in line with rest of file, isn't?

Reply:
sorry